src/types/log.rs

```diff
@@ -102,16 +105,26 @@ pub struct FilterBuilder {
 impl FilterBuilder {
     /// Sets from block
     pub fn from_block(mut self, block: BlockNumber) -> Self {
+        self.filter.block_hash = None;
         self.filter.from_block = Some(block);
         self
     }
 
     /// Sets to block
     pub fn to_block(mut self, block: BlockNumber) -> Self {
+        self.filter.block_hash = None;
         self.filter.to_block = Some(block);
         self
     }
 
+    /// Sets block hash
```

Comment:
Could you update documentation and mention that it's mutually exclusive with `from_block` and `to_block`?

Same with `from_block` and `to_block` docs.

Reply:
Sure!

Reply:
Thanks!